packages/atomic/src/components/facets-v1/atomic-rating-range-facet/atomic-rating-range-facet.tsx

```diff
@@ -0,0 +1,242 @@
+  @Prop() public icon = Star;
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+    this.initializeFacet();
+  }
+
+  private initializeFacet() {
+    const options: NumericFacetOptions = {
+      facetId: this.facetId,
+      field: this.field,
+      numberOfValues: this.numberOfIntervals,
+      currentValues: this.generateCurrentValues(),
+      sortCriteria: 'descending',
+      generateAutomaticRanges: false,
+    };
+    this.facet = buildNumericFacet(this.bindings.engine, {options});
+    this.facetId = this.facet.state.facetId;
+    this.bindings.store.state.numericFacets[this.facetId] = {
+      label: this.label,
+      format: (facetValue) => this.formatFacetValue(facetValue),
+    };
+  }
+
+  private get scaleFactor() {
+    return this.maxValueInIndex / this.numberOfIntervals;
+  }
+
+  private get numberOfSelectedValues() {
+    return this.facetState.values.filter(({state}) => state === 'selected')
+      .length;
+  }
+
+  private generateCurrentValues() {
+    const currentValues: NumericRangeRequest[] = [];
+    for (let i = 0; i < this.numberOfIntervals; i++) {
+      currentValues.push(
+        buildNumericRange({
+          start: Math.round((i + 1) * this.scaleFactor * 100) / 100,
+          end: Math.round(this.maxValueInIndex * 100) / 100,
+          endInclusive: true,
+        })
+      );
+    }
+    return currentValues;
+  }
+
+  private formatFacetValue(facetValue: NumericFacetValue) {
+    return this.bindings.i18n.t('to', {
+      start: facetValue.start,
+      end: facetValue.end,
+    });
+  }
+
+  private renderHeader() {
+    return (
+      <FacetHeader
+        i18n={this.bindings.i18n}
+        label={this.label}
+        onClearFilters={() => this.facet.deselectAll()}
+        numberOfSelectedValues={this.numberOfSelectedValues}
+        isCollapsed={this.isCollapsed}
+        onToggleCollapse={() => (this.isCollapsed = !this.isCollapsed)}
+      ></FacetHeader>
+    );
+  }
+
+  private renderValue(facetValue: NumericFacetValue, onClick: () => void) {
+    const displayValue = this.formatFacetValue(facetValue);
+    const isSelected = facetValue.state === 'selected';
+    return (
+      <FacetValueLink
+        displayValue={displayValue}
+        numberOfResults={facetValue.numberOfResults}
+        isSelected={isSelected}
+        i18n={this.bindings.i18n}
+        onClick={onClick}
+      >
+        <FacetValueIconRating
+          numberOfTotalIcons={this.maxValueInIndex}
+          numberOfActiveIcons={facetValue.start}
+          icon={this.icon}
+        ></FacetValueIconRating>
+        <span
+          class="ml-1 flex items-center group-hover:underline group-hover:text-primary"
+          style={
+            facetValue.start === this.maxValueInIndex
+              ? {fontWeight: 'bold'}
+              : {}
+          }
+          innerHTML={
+            facetValue.start === this.maxValueInIndex ? 'only' : '& up'
+          }
+        ></span>
```

Comment:
This could be moved from `innerHTML` to inside the span. E.g.:
```suggestion
        >
          {facetValue.start === this.maxValueInIndex ? 'only' : '& up'}
        </span>
```